### ISSUE TITLE
Broken srcs

### ISSUE THREAD

**Issue by Atemu** [NixOS/nixpkgs#104048]
**Describe the bug**

Tracking issue for srcs that can no longer be built with `nix-build '<nixpkgs>' -A pkgnamehere.src --check`.

Having these around isn't critical but it creates dependence on cache.nixos.org and it is not certain that it will keep around all srcs it once fetched.

**Comment by Atemu** [comment]
While reviewing #104045 I found a few:

```Nix
with import <nixpkgs> { }; [
  aacgain.src
  hybridreverb2.src
  m32edit.src
  x32edit.src
]

```

@robbinch
@magnetophon

**Comment by magnetophon** [comment]
Thanks for the heads-up @Atemu!
I found a fix for x32-edit and m32-edit and I'll do a PR for them asap.  (ie when I have a clean git state in my $NIXPKGS dir)

For hybridreverb2 I did not find the exact cause of problem though.
``nix-env`` complains: 
```
fatal: reference is not a tree: 78aec68237daa504c3ef87fc03e010695ace6684
Unable to checkout 78aec68237daa504c3ef87fc03e010695ace6684 from https://github.com/jpcima/JUCE.git.
```
When I look into the [.gitmodules](https://github.com/jpcima/HybridReverb2/blob/v2.1.1/.gitmodules), there is a pointer to https://github.com/jpcima/JUCE.git, but I don't understand why nix-env tries to download a non-existing reference.
I looked in various places for the 78ae... string, but didn't find it:  nixpkgs, hybridreverb2, JUCE.

Any pointers on why it tries to download this specific version of JUCE?
